src/help/zaphelp/index.xml

```diff
@@ -82,11 +84,15 @@
     <indexitem text="scan policy manager dialog" target="ui.dialogs.scanpolicymgr" />
     <indexitem text="search tab" target="ui.tabs.search" />
     <indexitem text="session contexts dialog" target="ui.dialogs.contexts" />
```

Comment:
The title of the target page now uses "context screens" instead of "contexts dialog", maybe this text should be changed?